packages/react-core/src/components/Dropdown/examples/DropdownSplitButtonProgressCheckbox.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import {
+  Dropdown,
+  DropdownToggle,
+  DropdownToggleCheckbox,
+  DropdownItem,
+  DropdownSeparator
+} from '@patternfly/react-core';
+
+export const DropdownSplitButtonProgressCheckbox: React.FunctionComponent = () => {
+  const [isOpen1, setIsOpen1] = React.useState(false);
+  const [isOpen2, setIsOpen2] = React.useState(false);
+
+  const [isChecked1, setIsChecked1] = React.useState(false);
+  const [isChecked2, setIsChecked2] = React.useState(false);
+
+  const onToggle1 = (isOpen: boolean) => {
+    setIsOpen1(isOpen);
+  };
+
+  const onToggle2 = (isOpen: boolean) => {
+    setIsOpen2(isOpen);
+  };
+
+  const onFocus = () => {
+    const element = document.getElementById('toggle-split-button-text');
```

Comment:
This is focusing the previous examples toggle button, and will end up causing the same dropdown toggle to receive focus regardless of which dropdown had a selection made in it.

Reply:
@jenny-s51 With the latest update, the only issue is that selecting an item in the first menu (without toggle text) will cause the toggle with toggle text to receive focus